projects/step-core/src/lib/modules/table/shared/table-remote-data-source.ts

```diff
@@ -110,12 +110,16 @@ const convertTableRequest = (req: TableRequestInternal): TableRequestData => {
 export class TableRemoteDataSource<T> implements TableDataSource<T> {
   private _terminator$ = new Subject<any>();
   private _inProgress$ = new BehaviorSubject<boolean>(false);
+  private _requestLock$ = new BehaviorSubject<boolean>(false);
```

Comment:
this subject does not seem to be completed in ngOnDestroy

Reply:
Good point, ngOnDestroy is commented out due to an issue but I added it there so we do not forget when commenting in again